Doc/library/subprocess.rst

```diff
@@ -1149,6 +1149,12 @@ calls these functions.
    code was zero then return, otherwise raise :exc:`CalledProcessError`. The
    :exc:`CalledProcessError` object will have the return code in the
    :attr:`~CalledProcessError.returncode` attribute.
+   .. note::
```

Comment:
You need a space between directives
```suggestion

   .. note::
```

Reply:
I don't know if this needs to be a note::. It's just a normal part of the doc. But I'm not a documentation expert.

Reply:
Oh right, thanks for pointing that out, I didn't realize that. Agreed. Let's just remove the entire `.. note::` block altogether.

Reply:
Changed according to the comments :-)